domains/paris.node.shing.json

```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "ShingSemicolon",
+    "email": "Shing.XD.0602@gmail.com"
+  },
+  "record": {
+    "A": ["141.145.205.47"],
```

Comment:
I cannot reach your website, keeps on loading

Reply:
It should work now.